### PR TITLE
Added support for Azure SSO with MSALv2; updated dependencies; added new code owner

### PR DESCRIPTION
**Pull request by herrkoch**
- By popular demand, @tbabik and I implemented login support for Azure SSO with MSALv2 for the Cypress quickstarter! 🥳
- We also updated all dependencies to the latest (compatible) versions.
- @tbabik also volunteered to be my successor as code owner for the Cypress quickstarter (Thanks!! 👍)

### CODE REVIEW

File: e2e-cypress/Jenkinsfile.template [hunk 1]
```diff
@@ -15,7 +15,24 @@ odsComponentPipeline(
 
 def stageTest(def context) {
   stage('Integration Test') {
-    withEnv(["TAGVERSION=${context.tagversion}", "NEXUS_HOST=${context.nexusHost}", "OPENSHIFT_PROJECT=${context.targetProject}", "OPENSHIFT_APP_DOMAIN=${context.getOpenshiftApplicationDomain()}"]) {
+  // OPTIONAL: load environment variables for Azure SSO with MSALv2; please adapt variable names to your OpenShift config
```
Comment (cschweikert) [inline e2e-cypress/Jenkinsfile.template:18]: nice 😎

File: e2e-cypress/files/README.md [hunk 2]
```diff
@@ -10,21 +10,28 @@ Please note that each stage is executed with its own cypress configuration file
 
 ## Running end-to-end tests
 
-Run `npm run e2e` to execute all end-to-end tests via [cypress](http://www.cypress.io) against the test instance of the front end.
+Run `npm run e2e` to execute all end-to-end tests via [cypress](http://www.cypress.io) against the test instance of the front end. (This includes a cleanup task called `delete-junit-results` which only runs in a UNIX-based shell. On a Windows command line you will probably get an error around `rm` not being found, please adapt `delete-unit-results` accordingly in that case.)
```
Comment (cschweikert) [inline e2e-cypress/files/README.md:13]: I once saw a little helper called ``rimraf`` (https://github.com/isaacs/rimraf). It is also available as npm package. When I remeber it correctly someone can use it **OS-independently**. After adding it to the dev dependencies (``npm install rimraf --save-dev``) it should be possible to use it like this:
```
  ...
  "delete-junit-results": "rimraf build/test-results``,
  ...
```

File: e2e-cypress/files/support/commands.ts [hunk 3]
```diff
@@ -7,6 +7,127 @@
 // https://on.cypress.io/custom-commands
 // ***********************************************
 
+// Azure SSO with MSALv2
+// ***********************************************
+// Originally developed by @tbabik and @herrkoch
+//
+// ATTENTION: This approach requires a technical, so-called "cloud-only" Azure user (<username>@<domain>.onmicrosoft.com) which does not
+// go through federation. (For Boehringer Ingelheim coders: Please refer to the "BI Coders Notebook", section "Cypress and Azure SSO" in
```
Comment (cschweikert) [inline e2e-cypress/files/support/commands.ts:15]: I would avoid any BI-specific remarks within ODS. Maybe write it like this:
"For security reasons those kind of users might require special adjustments within your organization. Please refer to your company's documentation around this topic or ask the IT department"

**Review by cschweikert — COMMENTED** [review]
Nice work! 🚀

File: e2e-cypress/files/support/index.ts [hunk 4]
```diff
@@ -16,5 +16,14 @@
 // Import commands.js using ES2015 syntax:
 import './commands';
 
+declare global {
+    namespace Cypress {
+        interface Chainable {
+            msalv2Login(): void;
+            crowdLogin(asTest): boolean;
```
[the remark this answers is not in the document]
Comment (herrkoch) [reply e2e-cypress/files/support/index.ts:23]: True! Thanks for the hint, will fix it!

File: e2e-cypress/files/support/commands.ts (hunk 3 above)
Comment (herrkoch) [reply e2e-cypress/files/support/commands.ts:15]: Agreed, we moved all of the BI-specific information out of the quickstarter but I wanted to leave this little hint in there so BI folks know where to look. :-) @michaelsauter said he would be fine with that. ;-)

File: e2e-cypress/Jenkinsfile.template [hunk 5]
```diff
@@ -15,7 +15,24 @@ odsComponentPipeline(
...
+  // sh "oc project <project-with-configured-secrets>"
+  // cypressUser = sh(returnStdout: true, script:"oc get secret e2euser -o yaml | grep USERNAME: | cut -d':' -f 2 | base64 -di")
+  // cypressPassword = sh(returnStdout: true, script:"oc get secret e2euser -o yaml | grep PASSWORD: | cut -d':' -f 2 | base64 -di")
+  // azureClientId = sh(returnStdout: true, script:"oc get secret azure -o yaml | grep AZURE_CLIENT_ID: | cut -d':' -f 2 | base64 -di")
+  // azureClientSecret = sh(returnStdout: true, script:"oc get secret azure -o yaml | grep AZURE_CLIENT_SECRET: | cut -d':' -f 2 | base64 -di")
+  // azureTenantId = sh(returnStdout: true, script:"oc get configmaps azure -o yaml | grep AZURE_TENANT: | cut -d':' -f 2 ").trim() // config map values are not base64 encoded
```
Comment (michaelsauter) [inline e2e-cypress/Jenkinsfile.template:24]: You can do this more directly by using `-ojsonpath`, e.g. `script:"oc get cm/azure -ojsonpath='{.data.AZURE_TENANT}'"`. Please test the exact syntax but this should do the trick.

File: e2e-cypress/files/README.md (hunk 2 above)
Comment (herrkoch) [reply e2e-cypress/files/README.md]: Thanks for the suggestion! I tested it on my Mac and on Jenkins, works fine on both so I included it.

File: e2e-cypress/Jenkinsfile.template (hunk 5 above)
Comment (herrkoch) [reply e2e-cypress/Jenkinsfile.template]: Thanks for the hint. I simplified and tested the statements based on your remark.

**Review by cschweikert — APPROVED** [review]

**Pull request merged**